File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/InvoiceFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.payments.Invoice;

public class InvoiceFilterValidator extends FilterValidator<Invoice> {

    public InvoiceFilterValidator(CustomValidator<Invoice> validator) {
        super(Invoice.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/InvoiceFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.Invoice
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class InvoiceFilterValidator(validator: Validator<Invoice>) : FilterValidator<Invoice>(
    Invoice::class, validator)
